CHANGELOG.md

```diff
@@ -1,7 +1,28 @@
 # Change Log
+
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
+## [5.1.0]
+
+### Added
+
+- Simplify the setup process with automated creation of deegree configurations and capabilities for WFS/WMS services.
+- Introduced support for the MS Access JDBC driver, expanding connectivity options for your database needs.
+- Migration to Java 17, updating Gradle Docker plugin to 9.3.1 and baseImage to Java 17, ensuring compatibility and leveraging the latest Java features.
+- Updated dependencies addressing security vulnerabilities, safeguarding your system against potential threats.
```

Comment:
I would omit this for two reasons:

1. There was not really anything done on this in this repository, this is mainly linked to the hale dependencies update and the Java update (which required updating the base image)
2. It's quite a bold statement with the "safeguarding", but the process on the hale studio side is not complete and there are some dependencies with vulnerabilities classified as critical